src/ir/branch-utils.h

```diff
@@ -97,6 +115,10 @@ void operateOnScopeNameUsesAndSentValues(Expression* expr, T func) {
       func(name, sw->value);
     } else if (auto* br = expr->dynCast<BrOn>()) {
       func(name, br->ref);
+    } else if (auto* res = expr->dynCast<Resume>()) {
+      // FIXME(frank-emrich) We can't really determine the values being sent
+      // here because they come from suspend instructions elsewhere. Is that a
+      // problem?
```

Comment:
New EH will have the same problem. I would just pass `nullptr` for now and see what breaks. Hopefully nothing!

Reply:
Thanks, that did work!